api/src/main/java/org/pmiops/workbench/utils/mappers/CommonMappers.java

```diff
@@ -34,6 +34,13 @@ public Long timestamp(Timestamp timestamp) {
     return null;
   }
 
+  public String timestampToString(Timestamp timestamp) {
```

Comment:
Mapstruct has a `toString` override, and it does the wrong thing. Or at least has different behavior than our current converter.

Reply:
That may actually work if [XmlGregorianCalendar](https://docs.oracle.com/javase/7/docs/api/javax/xml/datatype/XMLGregorianCalendar.html) has the requisite precision.

Reply:
Mainly it is different behavior from what we currently have. I would much rather use the same conversion method than what is already being used in the conversion, and clean up if we think we should later.

Reply:
Have you established that it's different functional behavior? Or just a different implementation? It would be neat to have a test case that runs through a bunch of timestamps to do the conversions.

The main advantage of MapStruct is that it makes the easy things trivial. If we don't trust it here, then it calls into question where we can trust it.

Reply:
I have just checked different implementation. I can check functional behavior.

Reply:
It is largely the same, with minor differences:
```
Gregorian: 2020-03-30T18:31:50.000Z
toString: 2020-03-30 18:31:50.0
```

I think gregorian is probably the right one to use, but should wait for the work on unifying date format.

Reply:
You're right; that's a functional change. Please put a comment with that difference in.